src/main/java/ru/aston/dao/MultitableDao.java

```diff
@@ -0,0 +1,12 @@
+package ru.aston.dao;
+
+import java.util.List;
+
+public interface MultitableDao<T, S, E> {
+
+    List<S> getProjectOfEmployee(E e);
+
+    List<T> getEmployeeOfProject(E e);
+
+    List<T> getEmployeeOfPosition(E e);
+}
```

Comment:
Unextandable. If you will need to add another entity you will need to extend generic